class-pinterest-for-woocommerce.php

```diff
@@ -252,11 +252,7 @@ public function init_plugin() {
 			// Disconnect advertiser if advertiser or tag change.
 			add_action( 'update_option_pinterest_for_woocommerce', array( $this, 'maybe_disconnect_advertiser' ), 10, 2 );
 
-			// Unschedule tasks if woocommerce is deactivated.
-			if ( defined( 'WC_PLUGIN_FILE' ) ) {
-				register_deactivation_hook( WC_PLUGIN_FILE, array( Pinterest\ProductSync::class, 'cancel_jobs' ) );
-			}
-
+			$this->maybe_update_plugin();
```

Comment:
@alexiglesias31 this looks out of place. (merge conflict resolution issue?)

Reply:
Yes, the line was added during the merge conflict resolution.